go_test.go

```diff
@@ -203,6 +203,23 @@ func TestGoPublishWithDeploymentView(t *testing.T) {
 	clientTestUtils.ChangeDirAndAssert(t, wd)
 }
 
+func TestGoPublishWithExclusion(t *testing.T) {
+	_, goCleanupFunc := initGoTest(t)
+	defer goCleanupFunc()
+	wd, err := os.Getwd()
+	assert.NoError(t, err, "Failed to get current dir")
+	prepareGoProject("project1", t, true)
+	jfrogCli := tests.NewJfrogCli(execMain, "jf", "")
+	err = execGo(jfrogCli, "gp", "v1.1.1", "--exclusions dir1/*")
```

Comment:
Unless I'm missing something, this test doesn't test the `exclusions` option, beycase `project1` doesn't seem to include a directory named `dir1`, and even if it, this test doesn't verify that dir1 has actually been excluded from the published archive.